.storybook/main.js

```diff
@@ -1,15 +1,13 @@
-const path = require('path');
+const path = require('path')
 
 module.exports = {
-  "stories": [
-    "../stories/**/*.stories.mdx",
-    "../stories/**/*.stories.@(js|jsx|ts|tsx)"
-  ],
-  "addons": [
-    "@storybook/addon-links",
-    "@storybook/addon-essentials",
+  stories: ['../stories/**/*.stories.mdx', '../stories/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
     '@storybook/addon-a11y',
-  // Handle SCSS modules
+    'storybook-addon-i18next/register',
```

Comment:
Note: This is actually the change to this file. Everything else is just auto-reformatting.

Reply:
thank you!

Reply:
I'm a bit suspect that this syntax doesn't match the rest, and I can't see the addon in my local storybook

Reply:
I suspect your comment might have been before you hard-re-installed, but in case it was because the syntax is weird. I did a little more work:

* Turns out that storybook addons are broken up into [3 categories](https://storybook.js.org/docs/react/configure/storybook-addons):
  * essential: ships with storybook
  * core: has the `@storybook` prefix and are located here https://github.com/storybookjs/storybook/tree/next/addons
  * community: everything else; doesn't necessarily follow a naming convention
* Also turns out that I don't need the "/register" portion of the string. This syntax was used before storybook switched over to [declarative config](https://storybook.js.org/blog/declarative-storybook-configuration/). I've removed it.